### PR TITLE
Update versions for Graduation

### PR DESCRIPTION
**Pull request by cjmctague**

### CODE REVIEW

**Comment by keith-turner** [comment]
Need to do this for recipes also.

**Comment by keith-turner** [comment]
We can also remove incubating from the SNAPSHOT version in the pom

**Comment by cjmctague** [comment]
@keith-turner I can do that for this PR if you would like

**Comment by keith-turner** [comment]
I think that makes sense to find them all in this PR

**Comment by keith-turner** [comment]
I ran the following

```
$ git clean -xfd
$ find . -type f  | fgrep -v .git | xargs grep -i incubat
```

Seeing poms and DISCLAIMER... Also seeing some URLs in other files, but we can not change those yet.

**Comment by keith-turner** [comment]
Its going to be so nice not having incubating in our versions.

**Comment by mikewalch** [comment]
I found one more incubating reference that should be removed:

```
./modules/core/pom.xml:  <name>Apache Fluo (incubating) Core</name>
```

**Comment by cjmctague** [comment]
Yea I noticed that a minutes ago. fixed ac0b1bc

File: pom.xml [hunk 1]
```diff
@@ -21,9 +21,9 @@
     <version>1-incubating</version>
```
Comment (cjmctague) [inline pom.xml]: Is this alright?

**Comment by cjmctague** [comment]
Also have to update `contrib/create-release-candidate.sh` to not include incubating. I'll leave that for one of you guys to do so I don't mess it up.

File: pom.xml (hunk 1 above)
Comment (mikewalch) [inline pom.xml]: Yes.  That is the latest released version of the parent pom.

**Comment by mikewalch** [comment]
Looks good to merge now

**Pull request merged**